plugins/source/aws/resources/services/fsx/data_repo_tasks.hcl

```diff
@@ -0,0 +1,48 @@
+//check-for-changes
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
+resource "aws" "fsx" "data_repo_tasks" {
+  path = "github.com/aws/aws-sdk-go-v2/service/fsx/types.DataRepositoryTask"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ServiceAccountRegionMultiplexer"
+    params = ["fsx"]
+  }
+  options {
+    primary_keys = ["arn"]
+  }
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSAccount"
+    }
+  }
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "resource_arn" {
+    rename = "arn"
+  }
+
+  column "tags" {
+    type = "json"
+    generate_resolver = true
```

Comment:
I'm not 100% sure it'll work in this case, but it seems like it should: did you consider using `client.ResolveTags` here? Like this:

```
    resolver "resolveTags" {
      path = "github.com/cloudquery/cloudquery/plugins/source/aws/client.ResolveTags"
    }
```

(instead of `generate_resolver = true`)

Reply:
Yes, it works, thanks!